library/Zend/View/Abstract.php

```diff
@@ -902,11 +902,11 @@ public function render($name)
     public function escape($var)
     {
         if (in_array($this->_escape, array('htmlspecialchars', 'htmlentities'))) {
-            return call_user_func($this->_escape, $var, ENT_COMPAT, $this->_encoding);
+            return call_user_func($this->_escape, "{$var}", ENT_COMPAT, $this->_encoding);
```

Comment:
Why are you passing an array to here instead of a string? Why not ensure your array is converted to a string prior to calling the function?